### DISCUSSION TITLE
How to migrate .Net framework WCF to CoreWCF?

### DISCUSSION THREAD

**Discussion by amarendra-k** [CoreWCF/CoreWCF#439] [Q&A] [answered]
Is there any tool for the migration? If not How to do?

**Comment by mconnew** [comment, marked as answer]
Look at the samples folder and there are .NET Framework and .NET Core equivalents of the same service. You can compare them to see how they are equivalent. There's also a basic primer in [this blog post](https://corewcf.github.io/blog/2021/02/19/corewcf-ga-release).
